### PR TITLE
style(reqtran) fix broken tables

### PR DESCRIPTION
**Pull request by h3xar0n**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Fixes table formatting in Request Transformer plugin that was causing unstyled tables and accidental header in navigation.

### Full changelog

* Add spaces after each table row
* Add | before each row in a table with an empty cell
* Mirror changes in all versions

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] Spellchecked my updates
- [ ] Ready to be merged

### CODE REVIEW

**Comment by h3xar0n** [comment]
When the preview is ready, check here: https://deploy-preview-1169--kongdocs.netlify.com/hub/kong-inc/request-transformer/0.1-x.html

**Comment by james-callahan** [comment]
The preview is still broken down below.

**Comment by h3xar0n** [comment]
@levfishbluefish I still need to fix some of the tables; they work in MD, but not on the page. @james-callahan pointed out there are some HTML tags that are still not rendering as list items.

**Comment by james-callahan** [comment]
Still a little weirdness. I see e.g. `\?q1=v1` in the output. I think that `\` there was originally escaping the `?`

File: app/_hub/kong-inc/request-transformer/index.md [hunk 1]
```diff
@@ -185,16 +220,36 @@ $ curl -X POST http://localhost:8001/services/example-service/plugins \
...
+        <li>h2: v1</li>
+        <li>h3: v1</li>
+      </ul>
+    </td>
+  </tr>
+</table>
 
-incoming request querystring | upstream proxied querystring
----           | ---
-\?q1=v1       | \?q1=v1&q2=v1
```
Comment (h3xar0n) [inline app/_hub/kong-inc/request-transformer/index.md]: @kikito , could you please let me know the intent of using `\?` in this particular table?

File: app/_hub/kong-inc/request-transformer/index.md [hunk 2]
```diff
@@ -185,16 +220,36 @@ $ curl -X POST http://localhost:8001/services/example-service/plugins \
...
--             | \?q1=v2&q2=v1
```
Comment (h3xar0n) [inline app/_hub/kong-inc/request-transformer/index.md]: @kikito also, is this row meant to be a new line in the row above?

**Comment by h3xar0n** [comment]
@james-callahan , just checked with @kikito , you are correct.

File: app/_hub/kong-inc/request-transformer/index.md (hunk 1 above)
Comment (h3xar0n) [reply app/_hub/kong-inc/request-transformer/index.md]: (as discussed in person, removing `\`)

File: app/_hub/kong-inc/request-transformer/index.md (hunk 2 above)
Comment (h3xar0n) [reply app/_hub/kong-inc/request-transformer/index.md]: (the row is meant to be empty, as the upstream column shows what happens with an empty request)

**Pull request merged**